### ISSUE TITLE
composer.json not found

### ISSUE THREAD

**Issue by cavo789** [Avnsh1111/laravel-docker-with-php8.2-apache-mysql#1]
Hello

I've tried your script (your readme.md step by step procedure is very clear 👍) and got an error due to the absence of the `composer.json` file.

Here is the output  
![image](https://github.com/Avnsh1111/laravel-docker-with-php8.2-apache-mysql/assets/355999/14fcf4d6-580a-4868-9ace-681314e622d6)

I'm an experienced user of Docker and when I've seen your repo, my idea was to give it to a new colleague in my team so she can start to play with Docker / MySQL / Laravel in just a very few steps.  This for learning.  

So, right now, it's not possible due to the absence of the `composer.json` file and because you can't start `composer install` in that situation.

**Comment by Avnsh1111** [comment]
Thanks for appreciating.
 
In that case, you need to put your project in `laravel` folder first then you can go with this.
Do let me know if you face any issues.

Thanks

**Comment by cavo789** [comment]
Oh damned, sorry, read too fast. I've thought using your solution will also install Laravel but no and this was clearly mentioned in step 3. We had to install Laravel manually. Mea culpa.

My need was to find a solution based on Docker which make the installation transparent too.  The easiest solution for a very beginner to start a Dockerized Laravel application from scratch. And to drop and restart at any time (sandboxes).

I was searching how to run a few statements like "curl ..../laravel.zip ..." to do this in an automated way (as a replacement for your step 3) but then I've found another solution : https://laravel.com/docs/10.x/installation#getting-started-on-linux

Running `curl -s https://laravel.build/example-app | bash` do what I was searching for (for my junior colleague).  Allow her to play with Docker / Laravel / MySQL without the need to first learn how to install everything.  Step by step ;-)

I'll recommend that one to her.

Have a nice day.
